docs/usage/exposureclasses.md

```diff
@@ -0,0 +1,92 @@
+# ExposureClasses
+
+The Gardener API server provides a cluster-scoped `ExposureClass` resource.
+This resource is used to allow exposing the control plane of a Shoot cluster in various network enviroments like restricted corporate networks, dmzs etc.
+
+### Background
+
+The `ExposureClass` resource is based on the concept for the `RuntimeClass` resource in Kubernetes.
+
+A `RuntimeClass` abstracts the installation of a certain container runtime (e.g. gVisor, kata containers) on all nodes or a subset of the nodes in a Kubernetes cluster.
+See [here](https://kubernetes.io/docs/concepts/containers/runtime-class/).
+
+In contrast an `ExposureClass` abstracts the ability to expose a Shoot clusters control plane in certain network enviroments (e.g. corporate networks, dmz, internet) on all Seeds or a subset of the Seeds.
+
+Example: `RuntimeClass` and `ExposureClass`
+
+```yaml
+apiVersion: node.k8s.io/v1
+kind: RuntimeClass
+metadata:
+  name: gvisor
+handler: gvisorconfig
+# scheduling:
+#   nodeSelector:
+#     env: prod
+---
+kind: ExposureClass
+metadata:
+  name: internet
+handler: internetconfig
+# scheduling:
+#   seedSelector:
+#     matchLabels:
+#       network/env: internet
+```
+
+Similar like `RuntimeClasses` also `ExposureClasses` define a `.handler` field which reflects the name reference for the corresponding CRI configuration of the `RuntimeClass` and the control plane exposure configuration for the `ExposureClass`.
+
+The CRI handler for `RuntimeClasses` is usually installed by an admistrator (e.g. via a DemonSet which install the corresponding container runtime on the nodes).
+The control plane exposure configuration is managed by the Gardenlet as it need to take care that the control plane is exposed accordingly.
```

Comment:
In the previous line you talk about an administrator creating the `RuntimeClass` resources. For `ExposureClass` resources it's same, right? But here you write "managed by Gardenlet" and I think this is confusing. Can you correct this?